Source/Core/getAbsoluteURL.js

```diff
@@ -0,0 +1,41 @@
+/*global define*/
+define([
+        '../ThirdParty/Uri',
+        './defaultValue',
+        './defined',
+        './DeveloperError'
+    ], function(
+        Uri,
+        defaultValue,
+        defined,
+        DeveloperError) {
+    "use strict";
+
+    /**
+     * Given a URI, returns the extension of the URI.
+     * @exports getAbsoluteURL
+     *
+     * @param {String} url The Url.
+     * @param {String} base The base.
```

Comment:
When a parameter is optional, the parameter name should be surrounded by square brackets, like:

`@param {String} [base] The base uri.`